src/components/diagrams/singleLineDiagram/position-diagram.js

```diff
@@ -33,8 +34,10 @@ import { useIntlRef, useSnackMessage } from '@gridsuite/commons-ui';
 import { Paper } from '@mui/material';
 import DiagramHeader from '../diagram-header';
 
-const customSldStyle = (theme) => {
-    return {
+const useStyles = makeStyles((theme) => ({
```

Comment:
I think it would be better to use a single-line-diagram-content instead, after its refactorization is done. At the moment, the new component is not usable enough to be used here, but in the future, this should be the case.